bait-files/src/test/java/com/google/sps/servlets/RequestServletTest.java

```diff
@@ -0,0 +1,83 @@
+// package com.google.sps.servlets;
```

Comment:
Is this meant to be a commented out test? What should be happening here?

Reply:
meant for that to be on a different branch - I've deleted it